lib/rubocop/ext/regexp_node.rb

```diff
@@ -40,6 +44,23 @@ def each_capture(named: ANY)
         self
       end
 
+      private
+
+      def with_interpolations_blanked
+        children.reject(&:regopt_type?).map do |child|
```

Comment:
There's only one `regopt` at the end, right? If so, `.reject(&:regopt_type?)` => `[0...-1]`.

Reply:
Good point - I don't think I'd realised that even for a Regexp without options the regopt node was still present. I think the original was better readability-wise, but the new version is probably fine with a comment 👍

Reply:
How about I add a `RegexpNode#content_nodes` to `rubocop-ast`?

Reply:
Hmm, my slight concern is that `RegexpNode#content` already exists and ignores interpolation (those with `begin_type?`) nodes, whereas `RegexpNode#content_nodes` would not ignore interpolation nodes. I'm not sure I have an alternative suggestion, though

Reply:
Yeah, I know. I would actually mark `content` as `# @deprecated`; I can't think of a single good use for that method.